### PR TITLE
jaeger-operator - custom-log-level #165

### LINKED ISSUE #165: [jaeger-operator] Cannot set log-level via helm chart
The default log level does not provide useful output for diagnosing problems with jaeger deployments.  The helm chart does not provide a way to set the `--log-level` command line parameter to increase operator log output.

### PR DESCRIPTION
**Pull request by yury-kozlov**
#### What this PR does
Specify custom log level for the operator.

#### Which issue this PR fixes

- fixes #165

#### Checklist

- [x] [DCO](https://github.com/jaegertracing/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Commits are [GPG signed](https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification)
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (`[jaeger]` or `[jaeger-operator]`)

### CODE REVIEW

File: charts/jaeger-operator/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.21.1
```
Comment (batazor) [inline charts/jaeger-operator/Chart.yaml:4]: I think it's more of a minor change `2.21.2`

**Comment by batazor** [comment]
@yury-kozlov 

It might be better to do it this way:
```yaml
# deployment.yaml

        env:
          {{- range $key, $value := .Values.env }}
          - name: {{ $key }}
            value: {{ $value | quote }}
          {{- end }}
          
# values.yaml

env:
  # Specifies log level for the operator:
  # logLevel:
```

File: charts/jaeger-operator/Chart.yaml (hunk 1 above)
Comment (yury-kozlov) [reply charts/jaeger-operator/Chart.yaml:4]: fixed to minor version

**Comment by yury-kozlov** [comment]
> @yury-kozlov
> 
> It might be better to do it this way:
> 
> ```yaml
> # deployment.yaml
> 
>         env:
>           - name: OPERATOR_NAME
>             value: {{ include "jaeger-operator.fullname" . | quote }}
>           {{- range $key, $value := .Values.env }}
>           - name: {{ $key }}
>             value: {{ $value | quote }}
>           {{- end }}
>           
> # values.yaml
> 
> env:
>   # Specifies log level for the operator:
>   # logLevel:
> ```

Sounds reasonable. Fixed to a more common name: `.Values.extraEnv`

File: charts/jaeger-operator/values.yaml [hunk 2]
```diff
@@ -36,6 +36,12 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name:
 
+# Specifies extra environment variables passed to the operator:
+extraEnv:
```
Comment (batazor) [inline charts/jaeger-operator/values.yaml:40]: You should probably specify the default value `extraEnv: []`

**Comment by yury-kozlov** [comment]
@batazor Thank you for the review!

**Pull request merged**